app/models/chargeback_container_image.rb

```diff
@@ -35,7 +35,7 @@ def self.build_results_for_report_ChargebackContainerImage(options)
                     Container.joins(:container_group).where('container_groups.container_project_id = ? or container_groups.old_container_project_id = ?', id, id)
                   end
 
-    @containers = @containers.includes(:container_project, :old_container_project, :container_image)
+    @containers = @containers.includes(:container_project, :old_container_project, :container_group, :container_image)
```

Comment:
>Consider explicit container.container_group.created_on in reporting code; it might(?) be clearer what it means, and might remind you to includes/preload the pods to avoid N+1 if you do this on many containers.

@cben @lpichler preloading is a good idea